react-native/react/common-adapters/button.js.flow

```diff
@@ -6,7 +6,9 @@ export type Props = {
   onClick: Function,
   label: string,
   style: Object,
-  primary: bool
+  primary: bool,
```

Comment:
Note to the above re index.desktop vs index. When i had just index it'd complain about not being able to find the file (since flow doesn't know about .desktop)?